recipes/google-cloud-sdk/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+
+cp -r * $outdir
+for FNAME in gcloud gsutil bq
+do
+    sed -i.bak 's$# <cloud-sdk-sh-preamble>$export CLOUDSDK_PYTHON=/opt/anaconda1anaconda2anaconda3/bin/python$' $outdir/bin/$FNAME
```

Comment:
Change this to `s|# <cloud-sdk-sh-preamble>|export CLOUDSDK_PYTHON=$PREFIX/bin/python|g`

Reply:
Thanks much for the review, updated.